src/plugins/telemetryTable/TelemetryTableRow.js

```diff
@@ -65,8 +65,10 @@ define([], function () {
         getCellLimitClasses() {
             if (!this.cellLimitClasses) {
                 this.cellLimitClasses = Object.values(this.columns).reduce((alarmStateMap, column) => {
-                    let limitEvaluation = this.limitEvaluator.evaluate(this.datum, column.getMetadatum());
-                    alarmStateMap[column.getKey()] = limitEvaluation && limitEvaluation.cssClass;
+                    if(!column.isUnit) {
+                        let limitEvaluation = this.limitEvaluator.evaluate(this.datum, column.getMetadatum());
+                        alarmStateMap[column.getKey()] = limitEvaluation && limitEvaluation.cssClass;
+                    }
 
```

Comment:
Just double checking to see whether or not you need to put an entry into the alarmStateMap regardless.

Reply:
Not sure on this one... maybe a question for @charlesh88 or @akhenry, I wasn't filtering out unit columns before and they were showing up with limit classes like their counterpart value, it seemed wrong to me so I removed it.